Backend/DataAccessLayer/Repositories/IResetRepository.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DataAccessLayer.Repositories
+{
+    interface IResetRepository
+    {
+        void AddResetID(string email, string resetID, DateTime expirationTime);
```

Comment:
method name AddResetID -> createPasswordReset
string resetID -> GUID passwordResetID
There is no email attribute associated without passwordReset object, do you mean GUID userID

Reply:
I used email because when the user clicks the link to start the password reset process, they have to supply an email to send the password reset link to. 

The resetID is then created but before being added to the database, the GUID of the user associated with that email is determined.

Reply:
That determination should be made at a higher layer. AddResetId should take a user GUID, not an email address. As you have it now, you're duplicating functionality from the User management repo.